starknetdata/feeder/feeder.go

```diff
@@ -275,15 +275,20 @@ func (f *Feeder) Class(ctx context.Context, classHash *felt.Felt) (core.Class, e
 
 	switch {
 	case response.V1 != nil:
-		return adaptCairo1Class(response.V1)
+		compiledClass, cErr := f.client.CompiledClassDefinition(ctx, classHash)
+		if cErr != nil {
+			return nil, cErr
+		}
+
+		return adaptCairo1Class(response.V1, compiledClass)
 	case response.V0 != nil:
 		return adaptCairo0Class(response.V0)
```

Comment:
Just curios how definitions worked in V0 ?

Reply:
I am not sure if this answers the question but the main difference is V0 classes contain Cairo Assembly whereas V1 classes contain Sierra programs. Sierra programs are eventually compiled to cairo assembly and that is what the compiledClass in this PR represents.